plugins/binance/plugin.go

```diff
@@ -0,0 +1,22 @@
+package binance
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/schema"
+)
+
+func New() schema.Plugin {
+	return schema.Plugin{
+		Name: "binance",
+		Platform: schema.PlatformInfo{
+			Name:     "Binance",
+			Homepage: sdk.URL("https://binance.com"), // TODO: Check if this is correct
```

Comment:
```suggestion
			Homepage: sdk.URL("https://binance.com"),
```

Reply:
Thanks for pointing out this, removed the comment